shelley/chain-and-ledger/shelley-spec-ledger-test/test/Test/Shelley/Spec/Ledger/Fees.hs

```diff
@@ -29,12 +40,12 @@ import Shelley.Spec.Ledger.API
     PoolCert (..),
     PoolParams (..),
     RewardAcnt (..),
-    SignKeyVRF,
+    -- SignKeyVRF,  -- Not sure we are importing the right SignKeyVRF
     Tx (..),
     TxBody (..),
     TxIn (..),
     TxOut (..),
-    VerKeyVRF,
+    -- VerKeyVRF,  -- or the right VerKeyVRF
```

Comment:
this one imported from API resolves to `VRF.VerKeyVRF (VRF (Crypto e))` (in Shelley.Spec.Ledger.Keys) - in this module you're using e.g. `aliceVRF:: forall v. VRFAlgorithm v => ... VRF.VerKeyVRF v ...` which does line up when parametrising by VRF - and the same for `SignKeyVRF`

Reply:
THat is good to know. I have removed the comments.